contrib/systemd/pganalyze-collector.service

```diff
@@ -10,8 +10,8 @@ ProtectSystem=full
 ProtectHome=true
 CapabilityBoundingSet=
 NoNewPrivileges=true
-MemoryLimit=256MB
-Restart=Always
+MemoryLimit=256m
```

Comment:
Most documentation I can find has the suffixes uppercase - should we do the same? (i.e. `256M`)